tests/ha/single_pod_restart/test_data_server_pod_restart.py

```diff
@@ -370,3 +409,106 @@ def test_pod_restart_node_nw_down(self):
             LOGGER.info("Step 10: IOs completed successfully.")
         LOGGER.info("COMPLETED: Verify IOs before and after data pod restart, "
                     "pod shutdown by making mgmt ip of worker node down")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45510")
+    def test_io_data_server_pod_restart(self):
+        """
+        Verify IO when any 1 data pod and any 1 server pod restart by replica method
+        """
+        LOGGER.info("STARTED: Verify IO when any 1 data pod and any 1 server pod restart by "
+                    "replica method")
+        LOGGER.info("STEP 1: Perform WRITEs/READs/Verify with variable object sizes")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        t_t = str(perf_counter_ns())
+        self.test_prefix = f'test-45510-{t_t}'
```

Comment:
I think we need to change this as s3bench logs takes test-xxxxx name for logs 
right @sonalk0209

Reply:
But this also creating logs? So i don't think we need to change this?

Reply:
No we can keep this as is and extract just "test-<num>" from any prefix for nfs use My only concern is why not use perf_counter_ns directly if u r assigning it for every prefix anyways